### DISCUSSION TITLE
Remove the arrow on option selected ?

### DISCUSSION THREAD

**Discussion by npalbrecht** [azeezat/react-native-select#29] [General]
Hello there,

Thanks for this amazing plug-in, it's very useful in my app and I'm enjoying playing with it ! 😄 
I've a simple question : is there a way to remove the arrow when an option is selected ? 
I can't find a way to do it and my aim is to have something like this when the option is selected : 

![Screenshot 2023-08-15 at 12 16 36](https://github.com/azeezat/react-native-select/assets/9021946/b93e8def-706e-4430-86ff-8baf77d30b6a)

Thanks guys !

**Comment by azeezat** [comment]
@npalbrecht Thanks for the feedback! We are glad you found it useful. 

There us a property called `dropdownIcon`. You can replace this with anything of choice and style it with `dropdownIconStyle`

I have put together an example for you below. you can find it in the [sandbox](https://azeezat.github.io/react-native-select/) (When you select the third Dropdown)

```js
      <DropdownSelect
            label="Border has been removed"
            placeholder="Select users"
            options={[
              { label: 'John Doe', value: '12' },
              { label: 'James Bond', value: '13' },
            ]}
            selectedValue={users}
            onValueChange={(itemValue: any) => setUsers(itemValue)}
            isSearchable
            primaryColor={'purple'}
            dropdownStyle={{
              borderWidth: 0, // To remove border, set borderWidth to 0
            }}
            dropdownIcon={
              users && (
                <View style={styles.outerCircle}>
                  <View style={styles.innerCircle} />
                </View>
              )
            }
            dropdownIconStyle={users && { top: 20, right: 15 }}
          />
```

**Comment by npalbrecht** [comment]
Thanks for you reply but I'm sorry for the missunderstood.
What I want to edit is this arrow and set it with the example in the OP : 

![Screenshot 2023-08-15 at 18 05 16](https://github.com/azeezat/react-native-select/assets/9021946/fe328f33-e97c-42d6-9e74-16d935d7f49b)

**Comment by azeezat** [comment]
I see, that functionality is currently unavailable. But we can add that in the coming releases

**Reply by azeezat** [reply]
@npalbrecht I just published a new release 1.1.3 on NPM to address your issue! Please let me know if this helps.

**Reply by azeezat** [reply]
Dropdowns 4 and 5 in the [sandbox](https://azeezat.github.io/react-native-select/) contains some examples

**Comment by azeezat** [comment]
![image](https://github.com/azeezat/react-native-select/assets/9849221/4921347c-3fc5-4a78-b9d8-dcef52509520)

**Comment by npalbrecht** [comment]
Oh wow, that's perfect ! Thank you very much for your speed ! :)
